quotecache/quotecache.go

```diff
@@ -27,26 +27,36 @@ func (q Quote) IsExpired() bool {
 	return time.Now().After(expiry)
 }
 
+var QuoteCache = make(map[string]map[string]Quote)
```

Comment:
Not really. It should stay as a pseudo private element of the quotecache.go file so people don't dig into it from outside.